integrations/optimizely/lib/index.js

```diff
@@ -47,10 +48,17 @@ var optimizelyContext = {
 Optimizely.prototype.initialize = function() {
   var self = this;
   // Flag source of integration (requested by Optimizely)
-  push({
-    type: 'integration',
-    OAuthClientId: '5360906403'
-  });
+  if (window.optimizelyEdge) {
+    pushEdge({
+      type: 'integration',
+      OAuthClientId: '5360906403'
+    });
+  } else {
+    push({
+      type: 'integration',
+      OAuthClientId: '5360906403'
+    });
+  }
```

Comment:
Interesting, this seems to be documented [here](https://docs.developers.optimizely.com/web/docs/technology-partners#section-registration), but was only ever active in _Classic_ Web (see [here](https://github.com/optimizely/optimizely/blob/0ff9c47d7924fc9e2985220d6ddd5a96294b4d15/src/www/js/client/api.js#L992-L1001) and [here](https://github.com/optimizely/optimizely/blob/784edb96f4bc5fbd149d34796676ba8b3e8f4a69/src/www/js/client/rum.js#L179-L180)) and hasn't been ported to X Web/Edge. Doesn't hurt I guess

Reply:
+1 cool, thanks I should've done some due diligence and also checked the codebase! I'll get rid of pushing this if it's edge

Reply:
No worries! I'm not actually sure what to do about this. We no longer have folks actively developing integrations, but maybe if we get back there one day, it might be useful to be collect this data.

I think we could go either way (continue to track, or not) as long as we do the same thing regardless of what mode we're in (Classic Web, X Web, Edge).